src/js/view/controls/components/volumetooltip.js

```diff
@@ -7,30 +7,51 @@ export default class VolumeTooltip extends Tooltip {
     constructor(_model, name, ariaText, svgIcons) {
         super(name, ariaText, true, svgIcons);
 
+        const localization = _model.get('localization');
+
         this._model = _model;
         this.volumeSlider = new Slider('jw-slider-volume jw-volume-tip', 'vertical');
         this.volumeSlider.setup();
-        this.volumeSlider.element().classList.remove('jw-background-color');
 
-        setAttribute(this.volumeSlider.element(), 'aria-label', this._model.get('localization').volumeSlider);
+        const volumeSliderElement = this.volumeSlider.element();
+        volumeSliderElement.classList.remove('jw-background-color');
+
+        setAttribute(volumeSliderElement, 'aria-label', localization.volumeSlider);
+        setAttribute(this.container, 'tabindex', '0');
 
-        this.addContent(this.volumeSlider.element());
+        this.addContent(volumeSliderElement);
 
         this.volumeSlider.on('update', function (evt) {
             this.trigger('update', evt);
             this.el.focus();
         }, this);
 
+        const closeTooltipHelper = () => {
+            this.closeTooltip();
+        };
+
+        const openTooltipHelper = () => {
+            this.openTooltip();
+        };
+
+        this.container.addEventListener('blur', closeTooltipHelper);
+        this.container.addEventListener('focus', openTooltipHelper);
+
         this.ui = new UI(this.el, { directSelect: true })
             .on('click enter', this.toggleValue, this)
             .on('tap', this.toggleOpenState, this)
-            .on('over focus', this.openTooltip, this)
-            .on('out blur', this.closeTooltip, this);
+            .on('over', this.openTooltip, this)
+            .on('out', this.closeTooltip, this);
 
         this._model.on('change:volume', this.onVolume, this);
     }
 
     toggleValue() {
         this.trigger('toggleValue');
     }
+
+    destroy() {
+        this.container.removeEventListener('blur', this.closeTooltipHelper);
```

Comment:
`this.closeTooltipHelper` would be undefined in this case, it's just a local var in the constructor. Same goes for `this.openTooltipHelper`

Reply:
>You should instead define them as functions outside of the constructor's scope

Na, they should be a `this` object. Otherwise those handlers will be shared by all instances of the class.